specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/networkVerifier.json

```diff
@@ -0,0 +1,1247 @@
+            "$ref": "./examples/VerifierWorkspacePut.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "VerifierWorkspaces"
+        ],
+        "summary": "Updates Verifier Workspace.",
+        "operationId": "VerifierWorkspaces_Update",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceName",
+            "description": "Workspace name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Verifier Workspace object to create/update.",
+            "schema": {
+              "$ref": "#/definitions/VerifierWorkspaceUpdate"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "VerifierWorkspace updated.",
+            "schema": {
+              "$ref": "#/definitions/VerifierWorkspace"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CommonErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "VerifierWorkspacePatch": {
+            "$ref": "./examples/VerifierWorkspacePatch.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "VerifierWorkspaces"
+        ],
+        "summary": "Deletes Verifier Workspace.",
+        "operationId": "VerifierWorkspaces_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./networkManager.json#/parameters/NetworkManagerNameParameter"
+          },
+          {
+            "in": "path",
+            "name": "workspaceName",
+            "description": "Workspace name.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9_.-]*$"
+          }
+        ],
+        "responses": {
+          "202": {
```

Comment:
>202 [](http://example.com/codeflow?start=11&length=3)

same here 

"headers": {
                            "Location": {
                                "description": "The URL of the resource used to check the status of the asynchronous operation.",
                                "type": "string"
                            }
                        }

Reply:
same as here https://github.com/Azure/azure-rest-api-specs/pull/30560/files#r1764369061